src/overlays/actors/ovl_En_Ik/z_en_ik.h

```diff
@@ -11,11 +11,11 @@ typedef void (*EnIkActionFunc)(struct EnIk*, GlobalContext*);
 #define IK_GET_FF(thisx) ((thisx)->params & 0xFF)
 
 typedef struct {
-    /* 0x00 */ Gfx* unk_00;
-    /* 0x04 */ Vec3f unk_04;
-    /* 0x10 */ Vec3f unk_10;
-    /* 0x1C */ Vec3s unk_1C;
-    /* 0x24 */ s32 unk_24;
+    /* 0x00 */ Gfx* ikEffectDList;
+    /* 0x04 */ Vec3f ikEffectWorldPos;
+    /* 0x10 */ Vec3f ikEffectVelocity;
+    /* 0x1C */ Vec3s ikEffectRot;
+    /* 0x24 */ s32 ikEffectEnable;
```

Comment:
You really don't need the prefixes on all of these.
```suggestion
    /* 0x00 */ Gfx* dList;
    /* 0x04 */ Vec3f pos;
    /* 0x10 */ Vec3f velocity;
    /* 0x1C */ Vec3s rot;
    /* 0x24 */ s32 enabled;
```